gluoncv/data/transforms/video.py

```diff
@@ -0,0 +1,346 @@
+# pylint: disable= missing-docstring
+"""Extended image transformations to video transformations."""
+from __future__ import division
+import random
+import numbers
+import numpy as np
+import cv2
```

Comment:
cv2 is not a required dep, use some tricks similar to this(https://github.com/dmlc/gluon-cv/blob/4549c34a6a93e9781dcc5069cac8d49f39908c60/gluoncv/utils/viz/image.py#L76) when it will be used in function/class. So import gluoncv won't fail when cv2 isn't installed.

Reply:
Thank you for pointing this out, already changed.